packages/express/src/handlers.ts

```diff
@@ -18,10 +18,10 @@ export function notFound ({ verbose = false } = {}): RequestHandler {
 }
 
 export type ErrorHandlerOptions = {
-  public?: string,
-  logger?: boolean|{ error?: (msg: any) => void, info?: (msg: any) => void },
-  html?: any,
-  json?: any
+  public?: string;
```

Comment:
Are commas and semicolons interchangeable here?

Reply:
Yes, that was purely stylistic change for consistency.